### ISSUE TITLE
[Q] (Your Question?) Why I get this in the main screen?

### ISSUE THREAD

**Issue by ssombra** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1093]
**Your question or details you need. Please describe.**
After the last upgrade, I get these symbols in the main screen. The symbols disappear as soon as I go to other screens. Just to clarify, I also tried with Acenotass icon set and I get the same, so it seems not to be icon related. 

**specify the device variant related to your question.**
TFT35V2.0, firmware 26.x Sep 14 202.

**Additional context**
Add any other context or screenshots about the question here.![image](https://user-images.githubusercontent.com/42944253/93404507-fa14e680-f84f-11ea-9578-96cccf9c09b5.jpeg)

**Comment by oldman4U** [comment]
Have you tried the standard TFT35 bmp's?

**Comment by ssombra** [comment]
> Have you tried the standard TFT35 bmp's?

Nope. Now I feel stupid :|
These icons worked flawlessly before the upgrade so I thought it was due to that. I'll try the standard ones later today.
thanks

**Comment by ssombra** [comment]
> Have you tried the standard TFT35 bmp's?

Same problem with standard BMPs. I re-flashed the firmware with the standard set and problem persists. 

![unnamed](https://user-images.githubusercontent.com/42944253/93487640-1ace5200-f8cb-11ea-9b85-a8125890b7be.jpg)

**Comment by oldman4U** [comment]
Hey. No reason to feel stupid.

 I have a E3 based TFT so not sure if this makes a difference.

ssombra <notifications@github.com> schrieb am Do. 17. Sep. 2020 um 16:50:

**Comment by ssombra** [comment]
I don't know. It actually works as expected, but those strange symbols on the screen annoy me.

**Comment by oldman4U** [comment]
I understand this.

Did you use a precompiled version or home brew?
What else do you have attached to the TFT besides the cables to the mainboard.
Have you changed something in config.ini?

I tested the precompiled version with unchanged config.ini on my TFT E3 v3, and I can not see those issues;-(

**Comment by ssombra** [comment]
> I understand this.
> 
> Did you use a precompiled version or home brew?
> What else do you have attached to the TFT besides the cables to the mainboard.
> Have you changed something in config.ini?
> 
> I tested the precompiled version with unchanged config.ini on my TFT E3 v3, and I can not see those issues;-(

I compiled the firmware to my needs. I have been doing it for a while and I re-flash from time to time to have the latest version available. The TFT doesn`t have anything connected except for the cable from the mainboard. Regarding the Config.ini, I adjusted it to match my changes in the firmware.

**Comment by oldman4U** [comment]
Can you try for a test the precompiled version and unchanged config.ini. Please

**Comment by ssombra** [comment]
> Can you try for a test the precompiled version and unchanged config.ini. Please

I will and let you know the results. Thanks

**Comment by oldman4U** [comment]
👍🏻

ssombra <notifications@github.com> schrieb am Do. 17. Sep. 2020 um 22:14:

**Comment by ssombra** [comment]
Good news. Before using the precompiled one I decided to try again with a custom version, so I downloaded the latest firmware available (yesterday), compiled it using my settings and flashed the TFT. It seems that the problem is gone, no strange symbols in the main screen and I can continue using the icon pack I have! 
I still don't know what the problem was because the settings are the same, but the important thing is that it works.
